pom.xml

```diff
@@ -37,7 +37,16 @@
 			<version>1.2.17</version>
 		</dependency>
 
-
+		<dependency>
+			<groupId>io.swagger.core.v3</groupId>
+			<artifactId>swagger-core</artifactId>
+			<version>2.1.7</version>
+		</dependency>
+		<dependency>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-jaxrs</artifactId>
+			<version>1.6.2</version>
+		</dependency>
 	</dependencies>
 	<build>
 		<plugins>
```

Comment:
I am missing here the plugin to generate the documentation. Could you please check it ?